language/ir-testsuite/tests/expressions/my_address_equality.mvir

```diff
@@ -0,0 +1,25 @@
+main() {
```

Comment:
Should this be included in this PR?

Reply:
This PR was a test of the bot. I added this as change so that it would be triggered if it was still active.